frontend/src/components/Widgets/CustomGroupedButtons.tsx

```diff
@@ -40,6 +40,7 @@ export default function GroupedButtons({handleClick, selectedOption, options, se
           aria-label="select merge strategy"
           aria-haspopup="menu"
           onClick={handleToggle}
+          disabled={options?.length > 0 ? false : true}
```

Comment:
hey @ddelpiano i only have one suggestion here to make the button looks better, for now it looks like this
<img width="240" alt="Screenshot 2024-02-09 at 2 54 16 PM" src="https://github.com/MetaCell/sckan-composer/assets/32265731/dbde7d3d-b267-495a-a96a-085391ebe9c1">

if we just make an edit for the options passed value to have an initial value as an empty array if it's undefined in here 
`export default function GroupedButtons({handleClick, selectedOption, options = [], selectedIndex, handleMenuItemClick , hasFormat = false, format, disabled}: any) {
`

and then add the disabled state for both ButtonGroup and the Arrow button like this 

` <ButtonGroup variant="contained" ref={anchorRef} disabled={options?.length <= 0 || disabled}>
        <Button onClick={handleClick}>{selectedOption}</Button>
        <Button
          size="small"
          aria-controls={open ? 'split-button-menu' : undefined}
          aria-expanded={open ? 'true' : undefined}
          aria-label="select merge strategy"
          aria-haspopup="menu"
          onClick={handleToggle}
          disabled={options?.length <= 0}
        >
          <ArrowDropDownIcon />
        </Button>
      </ButtonGroup>`

then the button when looks like this 
<img width="275" alt="Screenshot 2024-02-09 at 2 55 00 PM" src="https://github.com/MetaCell/sckan-composer/assets/32265731/7ff426ad-2ece-4c7a-a309-0d7fd626cc83">